string-manipulation-1/src/is-lower-cased.ts

```diff
@@ -0,0 +1,9 @@
+/* exported isLowerCased */
+function isLowerCased(word: string): any {
```

Comment:
That's correct, but you can simply avoid having to loop through each character:
word === word.toLowerCase();